src/Compiler/FSComp.txt

```diff
@@ -261,7 +261,7 @@ chkVariableUsedInInvalidWay,"The variable '%s' is used in an invalid way"
 428,chkInvalidCustAttrVal,"Invalid custom attribute value (not a constant or literal)"
 429,chkAttrHasAllowMultiFalse,"The attribute type '%s' has 'AllowMultiple=false'. Multiple instances of this attribute cannot be attached to a single language element."
 430,chkMemberUsedInInvalidWay,"The member '%s' is used in an invalid way. A use of '%s' has been inferred prior to its definition at or near '%s'. This is an invalid forward reference."
-431,chkNoByrefAsTopValue,"A byref typed value would be stored here. Top-level let-bound byref values are not permitted."
+431,chkNoByrefAsValRepr,"A byref typed value would be stored here. Top-level let-bound byref values are not permitted."
```

Comment:
reverted this one - the error message doesn't refer to "top" or "val repr" or anything so if we rename it, it should be to something that actually reflects the error message content.  Given we're not doing that, we can just leave as is.